prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/MA/MA-receiver-transform.yml

```diff
@@ -0,0 +1,68 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+constants:
+  PID: '/PATIENT_RESULT/PATIENT/PID'
+
+elements:
+
+  - name: xtn-use-code-from-system
+    condition: '%resource.system = "email"'
+    value: [ '"NET"' ]
+    hl7Spec: [ '%{hl7TelecomPath}-2' ]
+
+  - name: ma-name-of-the-sender-application-Namespace-Id
+    condition: true
+    value: [ '"CDC PRIME - Atlanta, Georgia (Dekalb)"' ]
+    hl7Spec: [ 'MSH-3-1' ]
+
+  - name: ma-sender-application-Universal-Id
+    condition: true
+    value: [ '"2.16.840.1.114222.4.1.237821"' ]
+    hl7Spec: [ 'MSH-3-2' ]
+
+  - name: ma-sender-application-Universal-Id-Type
+    condition: true
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-3-3' ]
+
+  - name: ma-name-of-the-receiving-application
+    condition: true
+    value: [ '"MA-MDPH"' ]
+    hl7Spec: [ 'MSH-5-1' ]
+
+  - name: ma-OID-of-the-receiving-application
+    condition: true
+    value: [ '"2.16.840.1.113883.19.3.2"' ]
+    hl7Spec: [ 'MSH-5-2' ]
+
+  - name: ma-indicates-that-5.2-is-an-OID
+    condition: true
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-5-3' ]
+
+  - name: ma-receiving-facility-namespace-ID
+    condition: true
+    value: [ '"MA-MDPH"' ]
+    hl7Spec: [ 'MSH-6-1' ]
+
+  - name: ma-receiving-facility-universal-ID
+    condition: true
+    value: [ '"2.16.840.1.113883.19.3.2"' ]
+    hl7Spec: [ 'MSH-6-2' ]
+
+  - name: ma-receiving-facility-universal-ID-Type
+    condition: true
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-6-3' ]
+
+  - name: ma-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '%{PID}-10-3' ]
+
+  - name: ma-suppress-Hl7-Fields
```

Comment:
why is this named "ma-suppress-HL7-Fields"? It doesn't appear to be suppressing anything?

Reply:
Thank you.  Good catch.